### DISCUSSION TITLE
Short mp3 loop function

### DISCUSSION THREAD

**Discussion by jhotovec** [pschatzmann/arduino-audio-tools#1786] [Q&A] [answered]
I have built the mp3 to Bluetooth player. I would like to add an option to loop a short 1-2 second file indefinitely. Can you give me a few pointers as to how to do this! 
I am not very good at object programming.

**Comment by pschatzmann** [comment, marked as answer]
That would not work with the player!
You can try to use the FileLoop with an EncodedAudioStream in the A2DP callback
